WorkflowUI/Sources/ViewControllerDescription/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if canImport(UIKit)
+
+    import UIKit
+
+    extension UpdateChildScreenViewController where Self: UIViewController {
```

Comment:
Why not put this directly on `extension UIViewController` and get rid of the protocol?

Reply:
This is needed because the method references `Self`, which you can't do in an extension on a concrete type like `UIViewController`. Blueprint [does the same dance, for example](https://github.com/square/Blueprint/blob/main/BlueprintUI/Sources/ViewDescription/ViewDescription.swift#L8).